.github/workflows/update-standard.yml

```diff
@@ -0,0 +1,31 @@
+name: update-standard
+
+on:
+  push:
+    paths:
+    - 'dicom_standard/*.py'
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Run Makefile
+        run: |
+          make clean -C dicom_standard
```

Comment:
I didn't know what `-C` did, nice!

Reply:
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun

Maybe using `working-directory` would be slightly cleaner, since we wouldn't have to repeat it for each make command.

Reply:
Good catch, I hadn't seen this!